src/main/resources/file-substitutions/Configuration.js

```diff
@@ -33,22 +33,35 @@ r(function () {
     document.title = title;
 });
 
-var match = window.location.toString().match(/(https?):\/\/([^-]*)-?cdn\.learn\.civilservice\.gov\.uk\/[^/]+\/([^/]+)\/([^/]+)\/.*$/);
-if (!match) {
+console.log('Configuration.js');
```

Comment:
Should this be here?

Reply:
One console log is kept in the js file deliberately so that we know that the updated js file is executing else it won't be possible to know whether the new js files are used or not.

Reply:
> Should this be here?

One console log is kept in the js file deliberately so that we know that the updated js file is executing else it won't be possible to know whether the new js files are used or not.